cfp_crawler/proposal/spiders/crawler.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import json
+from scrapy import signals
+
+
+class CrawlerSpider(scrapy.Spider):
+    name = 'crawler'
+    allowed_domains = ['in.pycon.org']
+    url = "https://in.pycon.org"
+    proposals = []
+    file = open("proposals.json", "w")
+    def start_requests(self):
+        yield scrapy.Request(self.url + "/cfp/2018/proposals", callback = self.parse)
+
+    def parse(self, response):
+        proposal_links = response.xpath("//h3[@class='proposal--title']/a/@href").extract()
+        for link in proposal_links:
+            yield scrapy.Request(self.url + link, callback = self.parseProposal)
+        
+        
+        
+    def parseProposal(self, response):
+        title = response.xpath("//h1[@class='proposal-title']/text()").extract()[0].strip()
+        author = response.xpath("//p[@class='text-muted']/small/b/text()").extract()[0].strip()
+        created_on = response.xpath("//p[@class='text-muted']/small/b/time/text()").extract()[0].strip()
+
+        section = response.xpath("//section[@class='col-sm-8 proposal-writeup']/div")
+        some_dic = {}
+        for div in section:
+            heading = div.xpath(".//h4[@class='heading']/b/text()").extract()[0]
+            data = self.format_data(div.xpath(".//text()").extract(), heading)
+            data = data[2:-2]
+            some_dic[heading[:-1]] = data
+        
+        table = response.xpath("//table/tr")
+        for col in table:
```

Comment:
And this should be `for row in table_rows`

Reply:
Done!